### ISSUE TITLE
Shuffle feels very "not random"

### ISSUE THREAD

**Issue by KeeganScott** [Soundnode/soundnode-app#710]
When I choose shuffle, I keep getting the same song 3 or 4 times within 10 songs.

**Comment by weblancaster** [comment]
Yeah the shuffle algorithm (if I can that) isn't a facy one.. although there is a PR to improve how the shuffle works just waiting the person fix it #684

**Comment by RedstoneGeek223** [comment]
...In order to make something seem more "random" you have to make the random generator less random. I'll let Vsause explain link => [https://www.youtube.com/watch?v=9rIy0xY99a0](url)
